src/ngraph/serializer.cpp

```diff
@@ -2370,7 +2371,11 @@ json JSONSerializer::serialize_node(const Node& n)
         node["ellipsis_mask"] = tmp->get_ellipsis_mask();
         break;
     }
-    case OP_TYPEID::Elu: { break;
+    case OP_TYPEID::Elu:
+    {
+        auto tmp = dynamic_cast<const op::Elu*>(&n);
+        node["alpha"] = tmp->get_alpha();
```

Comment:
Should we check that dynamic_cast do not fails and returns not nullptr?

Reply:
We don't check it in any operator.
I don't want to break file convention with it.
It is probably safe to assume that this dynamic_cast will be always successful.